### PR TITLE
Mortem gift pack

### PR DESCRIPTION
**Pull request by HentaiStorm**
## About The Pull Request

Gifts for the approaching Halloween
* Fix commits spam & remove mortem rust_g.dll

## Why It's Good For The Game

1. Add stutter with russian language
2. Remove checking `SSticker.current_state` in character's setup view = Instant dressing a character in setup
3. Refactoring `/obj/effect/overmap/ship/proc/handle_wraparound()` = Adds the ability to move the ship across space boundaries
4. Add `apply_sanity_effect` in `/datum/reagent/drink/affect_ingest` = Adds the ability to gain insight for desires by light drinks
5. Replace `"Vagabond"` by `ASSISTANT_TITLE` = More control over vagabonds in code
6. Add `is_hot` proc for rag = Corrects fuel ignition with a rag
7. Replace most important` usr << browse` by nanoui with utf8 = Some windows having support of russian language
8. Dress in loadout replaced by this valid `*/gray` subtype = Corrects to view dressing in loadout
9. Add `standing.color = belt.color` in `update_icons` = Corrects to view belt color
10. Refactoring kaiser `gas_attack` proc = Fix the Kaiser ability to make gas attack 
11. Add `desires.Cut()` in `finish_rest()` = Corrects getting rest to prevent getting a double rest
12. Add `organ.replaced(organ.get_limb())` = Corrects to valid reattach organ

## Changelog
:cl:
add: Add stutter with russian language
fix: Instant dressing a character in setup
add: Adds the ability to move the ship across space boundaries
add: Adds the ability to gain insight for desires by light drinks
code: More control over vagabonds in code
fix: Corrects fuel ignition with a rag
fix: Some windows have nanoui with russian language support
fix: Corrects to view belt color
add: Adds the Kaiser ability to make gas attack 
fix: Corrects getting rest to prevent getting a double rest
fix: Corrects to valid reattach organ
/:cl:

### CODE REVIEW

**Comment by HentaiStorm** [comment]
image
Тогда можно убрать и английский 'a'

**Comment by HentaiStorm** [comment]
image

Just convenient window :)

**Comment by SirRichardFrancis** [comment]
Так-то да, но это косяк очень старый и не твой, поэтому до него доёбываться не стал.

**Comment by HentaiStorm** [comment]
image
OMG, bug is feature image

**Comment by HentaiStorm** [comment]
image
12* and nope

**Review by Pink-Chink — APPROVED** [review]
Looks fine.

**Comment by SirRichardFrancis** [comment]
Or does it? There is mistakes pointed out in different iteration and it does not conform to contribution guidelines.

File: code/controllers/subsystems/jobs.dm [hunk 1]
```diff
@@ -582,5 +582,5 @@ SUBSYSTEM_DEF(job)
 /datum/controller/subsystem/job/proc/ShouldCreateRecords(var/title)
 	if(!title) return 0
 	var/datum/job/job = GetJob(title)
-	if(!job || job == "Vagabond") return 0
+	if(!job || job == ASSISTANT_TITLE) return 0
```
Comment (Pink-Chink) [inline code/controllers/subsystems/jobs.dm:585]:
```suggestion
	if(!job || job == ASSISTANT_TITLE)
		return 0
```

Comment (SirRichardFrancis) [reply code/controllers/subsystems/jobs.dm:585]:
```suggestion
	if(!job || job == ASSISTANT_TITLE)
		return
```

Comment (SirRichardFrancis) [reply code/controllers/subsystems/jobs.dm:585]:
```suggestion
	if(!job || job == ASSISTANT_TITLE)
		return FALSE
```

File: code/__DEFINES/misc.dm [hunk 2]
```diff
@@ -178,6 +178,8 @@
 
 
 #define LIST_OF_CONSONANT list("a", "b", "c", "d", "f", "g", "h", "j", "k", "l", "m", "n", "p", "q", "r", "s", "t", "v", "w", "x", "y", "z", "á", "â", "ã", "ä", "æ", "ç", "é", "ê", "ë", "ì", "í", "ï", "ð", "ñ", "ò", "ô", "õ", "ö", "÷", "ø", "ù")
+#define LIST_OF_CONSONANT_RU list("а","б","в","г","д","ж","з","й","к","л","м","н","п","р","с","т","ф","х","ц","ш","щ")
```
Comment (SirRichardFrancis) [inline code/__DEFINES/misc.dm:181]:
```suggestion
#define LIST_OF_CONSONANT list("b", "c", "d", "f", "g", "h", "j", "k", "l", "m", "n", "p", "q", "r", "s", "t", "v", "w", "x", "y", "z", "á", "â", "ã", "ä", "æ", "ç", "é", "ê", "ë", "ì", "í", "ï", "ð", "ñ", "ò", "ô", "õ", "ö", "÷", "ø", "ù")
#define LIST_OF_CONSONANT_RU list("б","в","г","д","ж","з","й","к","л","м","н","п","р","с","т","ф","х","ц","ш","щ")
```

**Pull request merged**